static/styles/compose.scss

```diff
@@ -67,7 +67,7 @@
 .compose_table {
     .message_header_colorblock {
         /* box-shadow: 0px 2px 3px hsl(0, 0%, 80%); */
-        box-shadow: inset 0px 2px 1px -2px hsl(0, 0%, 20%), inset 2px 0px 1px -2px hsl(0, 0%, 20%);
+        box-shadow: 0px 2px 1px -2px hsl(0, 0%, 20%) inset, 2px 0px 1px -2px hsl(0, 0%, 20%) inset, 0px -2px 1px -2px hsl(0, 0%, 20%) inset;
```

Comment:
@Bhanu-mbvg Thanks for adding the missing shadow. I thought it was some height related issue.